lib/core.js

```diff
@@ -0,0 +1,231 @@
+/*
+Copyright (c) 2014, Yahoo! Inc. All rights reserved.
+Copyrights licensed under the New BSD License.
+See the accompanying LICENSE file for terms.
+*/
+var __extends = (this && this.__extends) || (function () {
```

Comment:
is this going to be replicated in every file?

Reply:
unfortunately yeah, the other option is to use `tslib` but that comes w/ way more than we need, or make our own helper.
However, this dist is way smaller thou (old version was 5K, this is 2.75K)